src/content/docs/journey-performance/browser-diagnostics.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: "What's causing my app's high latency?"
+tags:
+- Learning paths
+- Customer experience
+- Browser monitoring
+- Latency
+metaDescription: "How to find, diagnose, and fix, an instance of your abnormal or poor app latency."
+signupBanner:
+  text: Monitor and improve your entire stack. 100GB free. Forever.
+---
+
+import solutionsOmaUprPatternNormal from 'images/solutions_screenshot-full_oma-upr-pattern-normal.png'
+
+import solutionsNormalPercentilePattern from 'images/solutions_screenshot-full_normal-percentile-pattern.png'
+
+import solutionsPatternAbnormal from 'images/solutions_screenshot-full_pattern-abnormal.png'
+
+import solutionsPatternAbnormalCompare from 'images/solutions_screenshot-full_pattern-abnormal-compare.png'
+
+Your app's performance is one of the most important aspects of whether or not it will succeed. [According to Google's research](https://think.storage.googleapis.com/images/micromoments-guide-to-winning-shift-to-mobile-download.pdf), 29% of mobile users will avoid using a site or an application if it's too slow or too difficult to find information, with 70% of those users doing so because the site is too slow. Diagnosing and fixing performance issues helps your organization attract new users while retaining existing ones, and New Relic is just the tool to help you do it. Once you've created <InlinePopover type="dashboards" /> for your services, you can use them to find issues causing high latency.
```

Comment:
```suggestion
Your app's performance is one of the most important aspects of whether or not your business will succeed. [According to Google's research](https://think.storage.googleapis.com/images/micromoments-guide-to-winning-shift-to-mobile-download.pdf), 29% of mobile users will avoid using a site or an application if it's too slow or too difficult to find information, with 70% of those users doing so because the site is too slow. Diagnosing and fixing performance issues helps your organization attract new users while retaining existing ones, and New Relic is just the tool to help you do it. Once you've created <InlinePopover type="dashboards" /> for your app's services, you can use them to find issues causing high latency.
```

Reply:
Looks better than what I had, thank you!